n2v/grid/grider.py

```diff
@@ -159,55 +164,51 @@ def generate_dft_grid(self, wfn):
 
         dft_grid = [x,y,z,w]
```

Comment:
Given that the output is already a numpy array. Why don't we replace line 150 - 153 with a dft_grd = np.empty((4, points)) and then inside loop just do grid_dft[0][offset...]
grid_dft[1][offset...]
And we save up a few lines.